src/plugins/intel_cpu/src/utils/profiler.hpp

```diff
@@ -0,0 +1,221 @@
+// Copyright (C) 2018-2022 Intel Corporation
```

Comment:
Do you plan to merge this profiler to master branch ?

Reply:
reverted